ui/raidboss/data/05-shb/raid/e5s.js

```diff
@@ -7,6 +7,206 @@
+      regexDe: Regexes.ability({ id: '4BC6', source: 'Ramuh', capture: false }),
+      regexFr: Regexes.ability({ id: '4BC6', source: 'Ramuh', capture: false }),
+      regexJa: Regexes.ability({ id: '4BC6', source: 'ラムウ', capture: false }),
+      regexKo: Regexes.ability({ id: '4BC6', source: '라무', capture: false }),
+      alarmText: function(data) {
+        if (data.leaderCount %2 == 0) {
+          return {
+            en: 'Spread',
+            de: 'Verteilen',
+            fr: 'Ecartez-vous',
+            ja: '散開',
+            cn: '分散',
+            ko: '산개',
+          };
+        }
+      },
+    },
+    {
+      id: 'E5S Crippling Blow',
+      regex: Regexes.ability({ id: '4BCA', source: 'Ramuh', capture: false }),
+      regexDe: Regexes.ability({ id: '4BCA', source: 'Ramuh', capture: false }),
+      regexFr: Regexes.ability({ id: '4BCA', source: 'Ramuh', capture: false }),
+      regexJa: Regexes.ability({ id: '4BCA', source: 'ラムウ', capture: false }),
+      regexKo: Regexes.ability({ id: '4BCA', source: '라무', capture: false }),
+      alertText: function(data, matches) {
+        if (matches.target == data.me) {
+          return {
+            en: 'Tank Buster on YOU',
+            de: 'Tankbuster auf DIR',
+            fr: 'Tank Buster sur VOUS',
+            ko: '탱버 대상자',
+          };
+        }
+      },
+    },
+    {
+      id: 'E5S Stormcloud Summons',
+      regex: Regexes.startsUsing({ id: '4BB8', source: 'Ramuh', capture: false }),
+      regexDe: Regexes.startsUsing({ id: '4BB8', source: 'Ramuh', capture: false }),
+      regexFr: Regexes.startsUsing({ id: '4BB8', source: 'Ramuh', capture: false }),
+      regexJa: Regexes.startsUsing({ id: '4BB8', source: 'ラムウ', capture: false }),
+      regexKo: Regexes.startsUsing({ id: '4BB8', source: '라무', capture: false }),
+      infoText: {
+        en: 'Position for Stormcloud',
+        ko: '번개 구름 위치',
+      },
+    },
+    {
+      id: 'E5S Centaur\'s Charge',
+      regex: Regexes.startsUsing({ id: '4BAD', source: 'Ramuh', capture: false }),
+      regexDe: Regexes.startsUsing({ id: '4BAD', source: 'Ramuh', capture: false }),
+      regexFr: Regexes.startsUsing({ id: '4BAD', source: 'Ramuh', capture: false }),
+      regexJa: Regexes.startsUsing({ id: '4BAD', source: 'ラムウ', capture: false }),
+      regexKo: Regexes.startsUsing({ id: '4BAD', source: '라무', capture: false }),
+      infoText: {
+        en: 'Be in your position',
+        ko: '자기 위치에 있기',
+      },
+    },
+    {
+      id: 'E5S Thunderstorm',
+      regex: Regexes.ability({ id: '4BBF', source: 'Ramuh', capture: false }),
+      regexDe: Regexes.ability({ id: '4BBF', source: 'Ramuh', capture: false }),
+      regexFr: Regexes.ability({ id: '4BBF', source: 'Ramuh', capture: false }),
+      regexJa: Regexes.ability({ id: '4BBF', source: 'ラムウ', capture: false }),
+      regexKo: Regexes.ability({ id: '4BBF', source: '라무', capture: false }),
+      infoText: {
+        en: 'Avoid AoE',
```

Comment:
I know you said this is just for melee, but I think this will be confusing to people who have to be hit by the thunderstorm.  Could you either adjust this to handle that case or remove it for now?

Reply:
This is not for melee only. Everyone should avoid aoes.

Reply:
Sorry for the confusion, I was mistaking this with the stormcloud abillity.

I think I'd say my feeling is that I think this trigger is not necessary, as it's not worth calling out obvious ground aoes.

Reply:
I was like so concentrated on dps-ing. forgot to avoid this, did jump and got hit by it lol.
After I become familiar with this, I was alright but as I experienced that, I put this trigger.
If you think this is not really needed, say it, I'll remove this.

Reply:
Yeah, my general rule of thumb is that visible ground aoes shouldn't be called out.  Things that are invisible or very fast (e.g. the cyclops in the new dungeon) are somewhat different.

This just sounds like drg problems.  :wink: